### PR TITLE
Feat #8639 UI : Show charts latest value in data insight overview card and rename datasets to data assets

### LINKED ISSUE #8639: UI : Show charts latest value in data insight overview card and rename datasets to data assets
- [ ] Show charts latest value in data insights overview card
- [ ] Rename datasets to data assets and entity to data asset

### PR DESCRIPTION
**Pull request by Sachin-chaurasiya**
Fixes #8639

### Describe your changes :

I worked on issue #8639, which includes,
- Showing overview of entities and web charts
- On click of overview scroll down to a particular chart
- Showing most active user with hover over the card

#
### Type of change :

- [x] New feature

#
### Frontend Preview (Screenshots) :

image

image

https://user-images.githubusercontent.com/59080942/201112686-a395f71f-2158-4d9d-b415-ecfe9875cf73.mov

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [x] I have commented on my code, particularly in hard-to-understand areas.
- [x] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#

### CODE REVIEW

File: openmetadata-ui/src/main/resources/ui/src/components/DataInsightDetail/DataInsightSummary.tsx [hunk 1]
```diff
@@ -11,51 +11,134 @@
...
+          dataInsightChartName: chartName,
+          dataReportIndex: DataReportIndex.EntityReportDataIndex,
+        };
+
+        return getAggregateChartData(params);
+      });
+
+      const responses = await Promise.allSettled(promises);
+
+      const chartDataList = responses.map((response) => {
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/components/DataInsightDetail/DataInsightSummary.tsx:84]: `chartDataList` will have undefined in the list can we remove them

File: openmetadata-ui/src/main/resources/ui/src/components/DataInsightDetail/DataInsightSummary.tsx [hunk 2]
```diff
@@ -11,51 +11,134 @@
...
+          dataInsightChartName: chart.chart,
+          dataReportIndex: chart.index,
+        };
+
+        return getAggregateChartData(params);
+      });
+
+      const responses = await Promise.allSettled(promises);
+
+      const chartDataList = responses.map((response) => {
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/components/DataInsightDetail/DataInsightSummary.tsx:133]: same as above

File: openmetadata-ui/src/main/resources/ui/src/styles/spacing.less [hunk 3]
```diff
@@ -397,3 +397,7 @@
 .m--ml-1 {
   margin-left: -0.25rem /* -4px */;
 }
+
+.m--ml-0\.5 {
+  margin-left: -0.125rem /* -2px */;
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/styles/spacing.less:402]: I really discourage to use negative margin or padding

**Review by chirag-madlani — COMMENTED** [review]
Can we also replace "No owner" with "--"

File: openmetadata-ui/src/main/resources/ui/src/styles/spacing.less (hunk 3 above)
Comment (Sachin-chaurasiya) [reply openmetadata-ui/src/main/resources/ui/src/styles/spacing.less:402]: @chirag-madlani, Sure, I'm using negative margin only if really needed.

**Comment by Sachin-chaurasiya** [comment]
@chirag-madlani , All comments are addressed and also updated the screenshots.

**Review by chirag-madlani — APPROVED** [review]

**Pull request merged**